perf/Project.toml

```diff
@@ -1,7 +1,9 @@
 [deps]
+Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
```

Comment:
Adapt and CUDA have to be in the perf environment now because they're required by `plot_helper.jl`, which is included by `amip_visualizer.jl` and `ncep_visualizer.jl`, which are included in `run_amip.jl` and then by `flame.jl` They also have to be in the test environment because `test/debug/debug_amip_plots.jl` includes `debug_plots.jl`, which includes `plot_helper.jl`